index.js

```diff
@@ -52,6 +53,7 @@ module.exports = function(config) {
     (!config.cloudwatchNamespace && config.cloudwatchStackname)) throw new Error('both cloudwatchNamespace and cloudwatchStackname must be configured');
 
   config.maxProcessTime = config.maxProcessTime || 3e5;
+  config.minProcessTime = config.minProcessTime || 0;
```

Comment:
@lily-chai @l-r what do you think about defaulting this to 200ms or 500ms?  

> [Each shard can support up to 5 transactions per second for reads, up to a maximum total data read rate of 2 MB per second.](https://docs.aws.amazon.com/streams/latest/dev/service-sizes-and-limits.html)

200ms  is as fast as we could read given no other consumers of the shard.